scripts/jenkins/ardana/ansible/init.yml

```diff
@@ -151,6 +151,26 @@
     tags:
       - create-ardana-input-model
 
+  - name: install patch
+    set_fact:
+        patches: {{ tempest_run_filter}}
+
+  - name: install patch
```

Comment:
I would use names like in the other tasks: uppercase initial letter and more descriptive
'Install "pacth" package'

For a moment I was expecting something else here X )